### ISSUE TITLE
Improve the survey experience via best design principles

### ISSUE THREAD

**Issue by reese-lee** [open-telemetry/sig-end-user#85]
I was inspired by [Pablo's post](https://cloud-native.slack.com/archives/C01RT3MSWGZ/p1732532153239329) to research survey design principles, as I'm sure there are ways to tweak our current survey experience that will produce better results overall (more respondents, higher quality responses, better experience for survey respondents--all of which will drive results for the original purpose of the survey). 

In my initial research, I found this [blog post](https://stripe.com/guides/atlas/survey-design-principles) that covers four (4) design principles, with some clear illustrated examples. 

There will be more work to do in terms of how we can ensure each survey meets these principles, but at the minimum we can work on the template as needed and the messaging (how we promote the survey). Thinking we can put together a similar principles-based guide that SIGs can follow for surveys. 

Thoughts? I'll build out the TODOs as appropriate after further discussion. 

Todos:
- [ ] TBD

**Comment by avillela** [comment]
Love this idea. Anything to entice folks to take our surveys!

**Comment by AndrejKiri** [comment]
Hi there, as I mentioned during the SIG call last week, I would like to work on this issue.

@reese-lee, you mentioned that the goal is to improve survey performance indicators, like response rate, quality of responses etc. Were there any problems with performance indicators of recent surveys?

I see that in the [README](https://github.com/AndrejKiri/sig-end-user/tree/main/end-user-surveys) of end-user-surveys folder there already is a list of _Community survey principles_ that was created by @danielgblanco and a _Basic survey template_ by @avillela. My understanding is that in this work we aim to improve them. Is it correct? What was your (anyone who created surveys) experience with using these guidelines and the template so far?

Any additional thoughts before I start working on it? 

Here is a simple TODO list proposal: 
- [ ] Compile a list of survey design principles resources
- [ ] Review recent community surveys and find common problems in survey design
- [ ] Suggest improvements to _Community survey design principles_ and _Basic survey template_

**Comment by AndrejKiri** [comment]
I saw that on LinkedIn, that we are using a URL shortener when sharing the survey link. This is awesome. 

These tools usually collect some data about people clicking on a link. Do we know how many people clicked on the link? This would allow us to calculate response rate, what is an important survey performance indicator. Also, do we use different links for different channels (e.g. LinkedIn, Slack)? 

<img width="545" alt="Image" src="https://github.com/user-attachments/assets/ee854713-1973-4554-849e-e5660aadc114" />

**Comment by danielgblanco** [comment]
> I see that in the [README](https://github.com/AndrejKiri/sig-end-user/tree/main/end-user-surveys) of end-user-surveys folder there already is a list of Community survey principles that was created by @danielgblanco and a Basic survey template by @avillela. My understanding is that in this work we aim to improve them. Is it correct?

Yeah, those docs were copied verbatim from the `community` repo and have not been reviewed/worked on in a long time.

**Comment by AndrejKiri** [comment]
For record, I brought up the usage of URL shorteners during a recent SIG meeting and proposed to create an issue to investigate it more. Here it is https://github.com/open-telemetry/sig-end-user/issues/95. 

Also, I learned that survey requests usually come from other SIGs. I think this is briefly mentioned in the SIG README, but there is no guidance for the folks from other SIGs on how to actually proceed. Based on @danielgblanco's suggestion, I created a survey issue template that could help with this in https://github.com/open-telemetry/sig-end-user/pull/93. I think it would make sense to include this information and reference the template also in the READMEs.

**Comment by AndrejKiri** [comment]
I created a PR with the guidelines – https://github.com/open-telemetry/sig-end-user/pull/105
